jetty-ee10/jetty-ee10-webapp/src/test/java/org/eclipse/jetty/ee10/webapp/StandardDescriptorProcessorTest.java

```diff
@@ -18,22 +18,43 @@
 
 import org.eclipse.jetty.server.Server;
 import org.eclipse.jetty.toolchain.test.MavenTestingUtils;
+import org.eclipse.jetty.util.resource.FileSystemPool;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.empty;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
 public class StandardDescriptorProcessorTest
 {
     //TODO add tests for other methods
-    
+    Server _server;
+
+    @BeforeEach
+    public void beforeEach() throws Exception
+    {
+        assertThat(FileSystemPool.INSTANCE.mounts(), empty());
+        _server = new Server();
+        _server.start();
+    }
+
+    @AfterEach
+    public void afterEach() throws Exception
+    {
+        _server.stop();
+        Configurations.getKnown().forEach(System.err::println);
```

Comment:
Please remove that `System.err::println` line.

Reply:
Done